### ISSUE TITLE
Request to be the Nightingale reviewer

### ISSUE THREAD

**Issue by chenjiandongx** [didi/nightingale#101]
***Request to be the Nightingale reviewer***

Hi, Nightingale-team.

@710leo @UlricQin. I have been keeping eyes on the nightingale project since it was published, for my current work stuff.

Thus I try to understand its design philosophy. During the source reading, I have sent more than **10 PRs** to nightingale, which include code refactor, bug fix, docs perfect and typo fix.

PRs list as below.

* #63
* #75
* #77
* #78
* #79
* #80
* #81
* #89
* #94
* #96
* #97
* #99
* #100

Nightingale is developed in open, hence I think it is possible that the developers who aren't working in DiDi could **have an opportunity to be one of the team members that has permission to create new branches and merge PR, etc.** That's why I file this issue for such a request.

In summary, I hope I can help your team to manage and perfect this project, making it better together.

**Comment by UlricQin** [comment]
@wbtiger

**Comment by chenjiandongx** [comment]
@UlricQin @710leo @wbtiger Thanks all you guys.
